app/Http/Controllers/FrontendTransportController.php

```diff
@@ -44,18 +44,20 @@ public function TrainStationboard(Request $request) {
 
     public function StationByCoordinates(Request $request)
     {
+        $request->validate([
```

Comment:
You have to save the result from ->validate in an variable to use the validated input.

for example: 
47: $validated = $request->validate([......
...
52: ....ates($validated['latitude'], $validated['longitude']);

Reply:
Sorry, my bad.